File: VehicleDBCoreProject/Model/VehicleModel.cs
using Vehicle.Model.Common;

namespace Vehicle.Model
{
    public class VehicleModel : IVehicleModel
    {
        public int VehicleModelId { get; set; }
        public string Name { get; set; }
        public string Abrv { get; set; }
        public int VehicleMakeId { get; set; }
    }
}

File: VehicleDBCoreProject/Vehicle.React/src/Components/ModelsAPI/VehicleModel.js
import React from 'react';

class VehicleModel extends React.Component {
	render() {
		return (
			<tr>
				<td>{this.props.model.vehicleMakeId}</td>
				<td>{this.props.model.name}</td>
				<td>{this.props.model.abrv}</td>
				<td><button onClick={this.props.editModel.bind(this, this.props.model.vehicleModelId)}>Edit</button></td>
				<td><button onClick={this.props.removeModel.bind(this, this.props.model.vehicleModelId)}>Remove</button></td>
			</tr>
		)

	}

}

export default VehicleModel;